crates/data_components/src/s3_vectors/put_vectors_sink.rs

```diff
@@ -0,0 +1,428 @@
+/*
+Copyright 2024-2025 The Spice.ai OSS Authors
```

Comment:
```suggestion
Copyright 2025 The Spice.ai OSS Authors
```

nit: for new files the copyright header should just be the current year